templates/compose/swetrix.yaml

```
@@ -0,0 +1,165 @@
+# documentation: https://docs.swetrix.com/selfhosting/how-to
+# slogan: Privacy-friendly and cookieless European web analytics alternative to Google Analytics.
+# category: analytics
+# tags: analytics,privacy,monitoring,open-source,clickhouse,redis
+# logo: svgs/swetrix.svg
+# port: 3000
+
+services:
+  swetrix:
+    image: swetrix/swetrix-fe:v4.0.2
+    restart: always
+    depends_on:
+      - swetrix-api
+    environment:
+      # Required - public URL of the API service (e.g. https://swetrix-api.example.com)
+      - API_URL=${API_URL:?}
+    healthcheck:
+      test:
+        - CMD-SHELL
+        - wget --no-verbose --tries=1 --spider http://localhost:3000/ping || exit 1
+      interval: 30s
+      start_period: 15s
+
+  swetrix-api:
+    image: swetrix/swetrix-api:v4.0.2
+    restart: always
+    environment:
+      # Required
+      - SECRET_KEY_BASE=${SECRET_KEY_BASE:?}
+
+      # Optional
+      - DISABLE_REGISTRATION=${DISABLE_REGISTRATION:-false}
+      - IP_GEOLOCATION_DB_PATH=${IP_GEOLOCATION_DB_PATH:-}
+      - DEBUG_MODE=${DEBUG_MODE:-false}
+      - CLOUDFLARE_PROXY_ENABLED=${CLOUDFLARE_PROXY_ENABLED:-false}
+
+      # SMTP (optional)
+      - SMTP_HOST=${SMTP_HOST:-}
+      - SMTP_PORT=${SMTP_PORT:-}
+      - SMTP_USER=${SMTP_USER:-}
+      - SMTP_PASSWORD=${SMTP_PASSWORD:-}
+      - FROM_EMAIL=${FROM_EMAIL:-}
+      - SMTP_MOCK=${SMTP_MOCK:-false}
+
+      # OIDC (optional)
+      - OIDC_ENABLED=${OIDC_ENABLED:-false}
+      - OIDC_ONLY_AUTH=${OIDC_ONLY_AUTH:-false}
+      - OIDC_DISCOVERY_URL=${OIDC_DISCOVERY_URL:-}
+      - OIDC_CLIENT_ID=${OIDC_CLIENT_ID:-}
+      - OIDC_CLIENT_SECRET=${OIDC_CLIENT_SECRET:-}
+
+      # Required service endpoints
+      - REDIS_HOST=redis
+      - CLICKHOUSE_HOST=http://clickhouse
+      - CLICKHOUSE_PASSWORD=${CLICKHOUSE_PASSWORD:-}
+    depends_on:
+      redis:
+        condition: service_healthy
+      clickhouse:
+        condition: service_healthy
+    healthcheck:
+      test:
+        - CMD-SHELL
+        - wget --no-verbose --tries=1 --spider http://localhost:5005/ping || exit 1
+      interval: 30s
+      start_period: 15s
```

Adding timeout and retries would be helpful

```suggestion
      timeout: 5s
      retries: 10
      interval: 30s
      start_period: 15s
```